src/express-lane-auction/IExpressLaneAuction.sol

```diff
@@ -0,0 +1,386 @@
+    address _beneficiarySetter;
+    /// @notice _roundTimingSetter The address given the rights to update the round timing info
+    address _roundTimingSetter;
+    /// @notice The admin that can manage all the admin roles in the contract
+    address _masterAdmin;
+}
+
+interface IExpressLaneAuction is IAccessControlEnumerableUpgradeable, IERC165Upgradeable {
+    /// @notice An account has deposited funds to be used for bidding in the auction
+    /// @param account The account that deposited funds
+    /// @param amount The amount deposited by that account
+    event Deposit(address indexed account, uint256 amount);
+
+    /// @notice An account has initiated a withdrawal of funds from the auction
+    /// @param account The account withdrawing the funds
+    /// @param withdrawalAmount The amount beind withdrawn
+    /// @param roundWithdrawable The round the funds will become withdrawable in
+    event WithdrawalInitiated(
+        address indexed account,
+        uint256 withdrawalAmount,
+        uint256 roundWithdrawable
+    );
+
+    /// @notice An account has finalized a withdrawal
+    /// @param account The account that finalized the withdrawal
+    /// @param withdrawalAmount The amount that was withdrawn
+    event WithdrawalFinalized(address indexed account, uint256 withdrawalAmount);
+
+    /// @notice An auction was resolved and a new express lane controller was set
+    /// @param isMultiBidAuction Whether there was more than one bid in the auction
+    /// @param round The round for which the rights were being auctioned
+    /// @param firstPriceBidder The bidder who won the auction
+    /// @param firstPriceExpressLaneController The address that will have express lane control in the specified round
+    /// @param firstPriceAmount The price in the winning bid
+    /// @param price The price paid by the winning bidder
+    /// @param roundStartTimestamp The time at which the round will start
+    /// @param roundEndTimestamp The time at which the round will end
+    event AuctionResolved(
+        bool indexed isMultiBidAuction,
+        uint64 round,
+        address indexed firstPriceBidder,
+        address indexed firstPriceExpressLaneController,
+        uint256 firstPriceAmount,
+        uint256 price,
+        uint64 roundStartTimestamp,
+        uint64 roundEndTimestamp
+    );
+
+    /// @notice A new express lane controller was set
+    /// @param round The round which the express lane controller will control
+    /// @param previousExpressLaneController The previous express lane controller
+    /// @param newExpressLaneController The new express lane controller
+    /// @param transferor The address that transferored the controller rights. The transferor if set, otherwise the express lane controller
+    /// @param startTimestamp The timestamp at which the new express lane controller takes over
+    /// @param endTimestamp The timestamp at which the new express lane controller will cease to have control
```

Comment:
for clarity, this should be the expectedEndTimestamp because ELC can be transferred

Reply:
I think the name is fine, have updated the comment